syft/execution/role.py

```diff
@@ -15,12 +15,19 @@
 from syft.execution.placeholder_id import PlaceholderId
 from syft.execution.state import State
 from syft.generic.frameworks.types import FrameworkTensor
+from syft.generic.tensor import AbstractTensor
 from syft.generic.object import AbstractObject
 from syft.generic.object_storage import ObjectStorage
 from syft.workers.abstract import AbstractWorker
 
 from syft_proto.execution.v1.role_pb2 import Role as RolePB
 
+from syft import dependency_check
+
+if dependency_check.crypten_available:
+    # TODO Crypten shouldn't be used here
+    import crypten
```

Comment:
This part can move into [the (newly created) list of `framework_packages`](https://github.com/OpenMined/PySyft/blob/master/syft/generic/frameworks/types.py#L27), which are available during the execution of `Plans`/`Roles`/`Actions` and used to execute package-level methods.